plugins/lb302/README

```diff
@@ -15,7 +15,7 @@ BUG:
   to be caused by 'unexhausted buffers' That is, the problem manifests
   itself to a greater degree when the user "buffer size" configuration
   is increased to over 1024 frames or so.  The problem is much less
-  noticable when the buffer size is set to 64 frames.
+  noticeable when the buffer size is set to 64 frames.
```

Comment:
I wouldn't spend too much time on old README files like this though. The rest of the info in there isn't up to date anyway.